lib/services/devices/deviceService.js

```diff
@@ -99,25 +100,34 @@ function createInitialEntityHandler(deviceData, newDevice, callback) {
  */
 function createInitialEntity(deviceData, newDevice, callback) {
     var options = {
-        url: 'http://' + config.getConfig().contextBroker.host + ':' + config.getConfig().contextBroker.port +
-            '/v1/updateContext',
-        method: 'POST',
-        json: {
-            contextElements: [
-                {
-                    type: deviceData.type,
-                    isPattern: 'false',
-                    id: String(deviceData.name),
-                    attributes: []
-                }
-            ],
-            updateAction: 'APPEND'
+            url: 'http://' + config.getConfig().contextBroker.host + ':' + config.getConfig().contextBroker.port +
+                '/v1/updateContext',
```

Comment:
The string `/v1/updateContext' is probably susceptible of being extracted to the `constants.js` file but I leave it totally to your discretion... ;)